app/controllers/course/condition/levels_controller.rb

```diff
@@ -0,0 +1,43 @@
+class Course::Condition::LevelsController < Course::ConditionsController
+  load_resource instance_name: :level_condition, class: Course::Condition::Level.name,
```

Comment:
`load_resource :level_condition, class: Course::Condition::Level.name` does that work? The first argument is the name to put the resource into.